eventapi.py

```diff
@@ -0,0 +1,32 @@
+import json
+from flask import request
+from flask_restful import Resource
+
+from event import Event
+
+def read_args(request):
+    return json.loads(request.data.decode('utf-8'))
+
+class EventAPI(Resource):
+    # Returns event details in JSON format
+    def get(self, name):
+        event = Event.get(name)
+        return event.serialize()
+
+    # Creates a new event based on args and name
+    def post(self, name):
```

Comment:
These methods should all implement a validation ruleset to ensure the data being save, retrieved, or updated is all valid